digital_land/phase/convert.py

```diff
@@ -143,6 +156,15 @@ def process(self, stream=None):
             # raise StopIteration()
             reader = iter(())
 
+        # save converted input to a file if custom_temp_dir and custom_temp_file are set
+        if hasattr(self, "custom_temp_dir") and hasattr(self, "custom_temp_file"):
```

Comment:
I would edit the ogr2ogr command (convert_features_to_csv) rather than something like this as otherwise you might end up making an extra copy of data which could impact large datasets a lot.